crates/ruma-common/Cargo.toml

```diff
@@ -14,21 +14,34 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
+# These feature gates exist only for the tests. Disabling them results in a
+# compile error.
```

Comment:
It's no longer so clear that this comment is about `client` and `server` in particular. Can you move them up right behind `default` and add an extra empty line after them?